boot/zephyr/CMakeLists.txt

```diff
@@ -168,8 +168,10 @@ if(CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256 OR CONFIG_BOOT_ENCRYPT_EC256)
     ${TINYCRYPT_DIR}/source/ecc.c
     ${TINYCRYPT_DIR}/source/ecc_dsa.c
     ${TINYCRYPT_DIR}/source/sha256.c
-    ${TINYCRYPT_DIR}/source/utils.c
     )
+    if(NOT CONFIG_ZEPHYR_TINYCRYPT_MODULE)
+      zephyr_library_sources(${TINYCRYPT_DIR}/source/utils.c)
+    endif()
```

Comment:
this shouldn't be here, you do not enable tinycrypt, you just use the specific bootloader tinycrypt option

Reply:
Do you mean
`CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256` and `CONFIG_BOOT_ECDSA_TINYCRYPT`?

Reply:
Yes